manual/basic_usage.md

```diff
@@ -71,6 +71,16 @@ $ rubocop -a
 
 See [Auto-correct](auto_correct.md).
 
+#### Changing what RuboCop considers to be offenses
+
+RuboCop comes with a preconfigured set of rules for each of its cops. Depending on your project, you may wish to
```

Comment:
I guess you might mention that the default rules are on https://rubystyle.guide.